test/mocha/gilded-rose.spec.ts

```diff
@@ -4,64 +4,88 @@ import { Item, GildedRose } from '@/gilded-rose';
 describe('Gilded Rose', () => {
   it('should degrade quality by one if sellin > 0', () => {
     const glidedRose = new GildedRose([new Item('item', 1, 5)]);
+
     const items = glidedRose.updateQuality();
+
     expect(items[0].quality).to.equal(4);
   });
   it('should degrade quality twice as fast if sellin <= 0', () => {
     const glidedRose = new GildedRose([new Item('item', 0, 5)]);
+
     const items = glidedRose.updateQuality();
+
     expect(items[0].quality).to.equal(3);
   });
   it('should not degrade anymore the quality if zero', () => {
     const glidedRose = new GildedRose([new Item('item', 0, 0)]);
+
     const items = glidedRose.updateQuality();
+
     expect(items[0].quality).to.equal(0);
   });
   it('aged brie should increse in quality by 1 if sellin > 0', () => {
```

Comment:
you could group the items specific tests by using nested `describe`

See https://stackoverflow.com/questions/19298118/what-is-the-role-of-describe-in-mocha